conduit/Data/Conduit/List.hs

```diff
@@ -631,6 +633,25 @@ consumeC =
 {-# INLINE consumeC #-}
 STREAMING0(consume, consumeC, consumeS)
 
+-- | Group a stream into chunks of a given size. The last chunk may contain
+-- fewer than n elements.
+--
+-- Subject to fusion
+--
+-- Since ???
```

Comment:
Can you put this at 1.2.9, and adjust the .cabal file and ChangeLog accordingly?